app/src/test/java/edu/uc/group/rankine/EditingSetTest.kt

```diff
@@ -1,7 +1,6 @@
 package edu.uc.group.rankine
 
 import edu.uc.group.rankine.dto.ElementObject
-import edu.uc.group.rankine.dto.FieldObject
 import edu.uc.group.rankine.dto.ObjectSet
 import org.junit.Test
 
```

Comment:
Removed import for FieldObject that did not exist, allowing unit test to pass.

Reply:
Are you sure these are related?  An import should not impact a unit test passing or failing.